deploy/handler/operator.yaml

```diff
@@ -204,6 +212,10 @@ spec:
           imagePullPolicy: {{ .HandlerPullPolicy }}
           command:
             - manager
+          resources:
+            requests:
+              cpu: "200m"
```

Comment:
This means that we will consume 20 percent of a CPU at all times. I wonder if we could be more conservative here while still offering good service

Reply:
from what i understand and checked (kubectl top pod) unless i am wrong
we don't consume it all the time, the cpu request also determines the share of the cpu and then the pods are "nice" to each other, but please correct me if i am wrong

I can check and reduce it to 150m if it will work good enough,
50m took 3 times the regular 2 minutes that it takes without limits
it takes 2 minutes minimum which is pretty long anyhow

150 would be fine ? or to try 100m / 120m even if it hit performance ?

Reply:
Keep me honest here, but would not that mean that idle knmstate is occupying space that could've been used by workload?

I think that even 50m should do as long as the configuration then succeeds... The point is that it would remain operational and won't be insanely slow. 3 times the time does not sound that terrible, especially if it allows for better use of the cluster. Setting these values too high may increase requirements of the whole HCO by too much.

Let's start with something between 50 and 150 if possible.

Reply:
by space i guess you meant CPU and not RAM, since this is the discussion
from what i saw, i am not sure it takes it on idle,
anyhow i will ask that offline to verify, and meanwhile will try to check 50 - 150m

Reply:
reduced to 100m (operates as good as without limits)
unless you really want below it

Reply:
Thanks sounds like a good start